partitioner/src/main/java/io/confluent/connect/storage/partitioner/TimeBasedPartitioner.java

```diff
@@ -132,7 +132,19 @@ public void configure(Map<String, Object> config) {
 
   @Override
   public String encodePartition(SinkRecord sinkRecord) {
-    long timestamp = timestampExtractor.extract(sinkRecord);
+    Long timestamp = timestampExtractor.extract(sinkRecord);
+    if (timestamp == null) {
+      String timestampExtractorClassName = timestampExtractor.getClass().getName();
+      log.error(
```

Comment:
nit : its the same message that's used for logger and the exception. May be we could create an error string and use it in both.

Reply:
Fixed